.github/workflows/run-unit-tests.yaml

```
@@ -0,0 +1,61 @@
+                "packages-up-to": ["husarion_ugv"],
+                "cmake-args": ["-DCMAKE_BUILD_TYPE=Release"]
+              },
+              "test": {
+                "packages-up-to": ["husarion_ugv"]
+              }
+            }
+
+        continue-on-error: true
+
+      - name: Retry Build and test (if failed)
+        if: steps.build_test_1.outcome == 'failure'
```

actionlint (1.7.4)

property "build_test_1" is not defined in object type {}

(expression)